catalogue/aarch64/tests/LB+rel+CAS-no-RnM-addr.litmus

```diff
@@ -0,0 +1,17 @@
+AArch64 LB+CAS-no-RnM-addr+rel
+
+{
+  0: X0 = x;      1: X0 = x;
+  0: X1 = y;      1: X1 = y;
+}
+
+P0              | P1                          ;
+LDR W3, [X0]    | LDR W3, [X1]                ;
+MOV W4, #1      | EOR W4, W3, W3              ;
+STLR W4, [X1]   | SXTW X4, W4                 ;
+                | ADD X0, X0, X4              ;
```

Comment:
litmus7 complains about this tests and all other tests that use SXTW a way around it would be to do:
```suggestion
STLR W4, [X1]   | ADD X0, X0, W4, SXTW        ;
```